docs/_includes/footer.html

```diff
@@ -10,6 +10,7 @@ <h3>Community contact</h3>
                     <a class="text-white" href="https://freenode.net/">Freenode</a> or 
                     <a class="text-white" href="https://lists.linuxfoundation.org/mailman/listinfo/grimoirelab-discussions">GrimoireLab mailing list</a> 
                     to get in touch with the <em>community</em>.</p>
+                    <p class="text-justify">Make sure to check out the <a href="">Contribution Guidelines</a> before contributing.</p>
```

Comment:
Please add the corresponding link (https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md) on the `href` field.

Reply:
Thanks for pointing it out! I've added the href value in the anchor tag.